.gitignore

```diff
@@ -9,3 +9,4 @@ archive_test
 index_test.html
 .idea/workspace.xml
 package-lock.json
+.gitpod.yml
```

Comment:
Hi @coding-dynasty, Why did you add this `.gitpod.yml` in this commit. Could you remove it recommit so that we can merge your PR. Thx.